packages/core/src/main/utils/resolve-system-proxy/resolve-system-proxy-window.injectable.ts

```diff
@@ -4,13 +4,21 @@
  */
 import { getInjectable } from "@ogre-tools/injectable";
 import { BrowserWindow } from "electron";
+import electronAppInjectable from "../../electron-app/electron-app.injectable";
 
 const resolveSystemProxyWindowInjectable = getInjectable({
   id: "resolve-system-proxy-window",
-  instantiate: () => {
-    const window = new BrowserWindow({ show: false });
+  instantiate: async (di) => {
+    const app = di.inject(electronAppInjectable);
 
-    window.hide();
+    await app.whenReady();
+    
+    const window = new BrowserWindow({
```

Comment:
This is a competing way to create new "lensWindows". That means it isn't recognised in eg.
`closeAllWindowsInjectable`, and doesn't therefore get closed. Maybe that's one source of this bug?

Reply:
I agree that we probably should close the window by ourselves but this particular window is not really like “normal” LensWindow .. and I even had a runnable for doing this but it did not help.

Reply:
This will mean that behaviour or name of `closeAllWindowsInjectable` needs to change, as now it's not accurate.

Resolved.